templates/cc/message.go

```diff
@@ -3,7 +3,7 @@ package cc
 const messageTpl = `
 	{{ $f := .Field }}{{ $r := .Rules }}
 	{{ template "required" . }}
```

Comment:
do we need to change java/required.go?

Reply:
Discussed offline that MessageRules would be attached to the Rules and so, can be accessed using `.Rules.GetRequired`